.github/workflows/manual_system_tests.yml

```diff
@@ -229,7 +229,6 @@ jobs:
       if: ${{ inputs.skip_tests == 'FALSE' }}
       run: |
         ./get_tag.sh 0chain ${{ inputs.miner_branch }} miner
```

Comment:
This is a temporary fix for backwards compatibility 
Once we are happy, we can rename "miner_branch" to "zerochain_branch" in all repos

Reply:
I reviewed the changes in system tests which depend this workflow again. I can implement the needed logic without these breaking changes